src/Umbraco.Infrastructure/Compose/NotificationsComposer.cs

```diff
@@ -62,6 +63,13 @@ public override void Compose(IUmbracoBuilder builder)
                 .AddNotificationHandler<ContentPublishedNotification, RedirectTrackingHandler>()
                 .AddNotificationHandler<ContentMovingNotification, RedirectTrackingHandler>()
                 .AddNotificationHandler<ContentMovedNotification, RedirectTrackingHandler>();
+
```

Comment:
This is fine for now, but later we should get rid of NotificationsComposer and just make all of this into an extension method on IUmbracoBuilder and call it explicitly.